### PR TITLE
ENCD-5151-fix-software-breadcrump

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

**Review by emro — DISMISSED** [review]
Looks good! You checked that the BDD tests passed locally?

**Comment by emro** [comment]
It sounds like there might be a fix for the BDD tests not passing, maybe it is worth waiting for that fix and re-running the BDD tests?

**Comment by zoldello** [comment]
@emro Good point. Since this is not going in the current spring AFAIK, it should be okay to wait until Paul's bdd-fix is merged

**Review by emro — APPROVED** [review]

**Pull request merged**
